src/PackingList/PackingList.js

```diff
@@ -6,16 +6,14 @@ import { connect } from 'react-redux'
 const PackingList = ({ packingList }) => {
 
   const { tripDetails, categories } = packingList
-  const { title, destination, number_of_days } = tripDetails
-
   const verifyPackingList = () => {
     if (packingList.categories) {
       return (
         <>
           <header className='packing-list-header'>
-            <h1>{title}</h1>
-            <h2>{destination}</h2>
-            <h3>{number_of_days} days</h3>
+            <h1>{tripDetails.title}</h1>
+            <h2>{tripDetails.destination}</h2>
+            <h3>{tripDetails.number_of_days} days</h3>
           </header>
           <button className='save-list-button'>
```

Comment:
We could refactor this button to toggle disabled on whether or not the form is filled out then error handle form there. (See Issue #16)